content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
-
-Once in the spreadsheet view, click on :guilabel:`File`, then select :guilabel:`Save` to save the
-data, or :guilabel:`Save as Template` to save the setting as a template to use in the future.
+   :alt: The view of data sent to a spreadsheet.
 
 .. note::
-   The *Documents* application must be installed in order to use the :guilabel:`Insert in
-   Spreadsheet` option.
+   If the *Documents* app is **not** installed, the :guilabel:`Insert in Spreadsheet` option places
+   the newly created spreadsheet in the *Dashboards* app.
 
+   If the *Documents* application **is** installed, the spreadsheet has the option to be stored in
+   either the *Dashboards* app or *Documents* app.
 
-Meal Vouchers
--------------
+Work entry analysis
+-------------------
 
-:guilabel:`Meal Vouchers` provides an overview of the meal vouchers used by employees, and can be
-shown by :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, :guilabel:`Quarter`, or
-:guilabel:`Year`. The default view is by month.
+The default :guilabel:`Work entry analysis` report provides an overview of the validated work
+entries for the current month. To view this report, navigate to :menuselection:`Payroll app -->
+Reporting -> Work Entry Analysis`.
 
-.. image:: reporting/meal-vouchers.png
-   :align: center
-   :alt: Meal voucher
+To change the displayed view, click the :guilabel:`➕ (plus)` icon next to the word
+:guilabel:`Total`, then click on one of the grouping options. The available options are
+:guilabel:`Work Entry Type`, :guilabel:`Employee`, and :guilabel:`Department`. If in a multi-company
+database, a :guilabel:`Company` option also appears.
 
-To change the displayed view, click on :guilabel:`Total`. The data collapses, showing only the count
-column. Click on :guilabel:`Total` again, then hover over :guilabel:`Day`, then click on one of the
-other time-period options available.
+To add a new group to sort the data by, click :guilabel:`Add Custom Group`, then click one of the
+presented options.
 
-.. image:: reporting/voucher-day.png
-   :align: center
-   :alt: Meal voucher
+.. tip::
+   Wherever a :guilabel:`➕ (plus)` icon appears on a pivot table, the information can be further
+   grouped. Click on a :guilabel:`➕ (plus)` icon to reveal the available grouping options.
 
-It is possible to compare the current meal voucher report to the one for the previous time period or
-the previous year. To view these comparisons, click on the :guilabel:`Comparison` drop-down menu at
-the top, then select either :guilabel:`Date: Previous Period` or :guilabel:`Date: Previous Year`.
+   Click on a :guilabel:`➖ (minus)` icon anywhere on the pivot table to remove that grouping.
 
-.. image:: reporting/meal-comparison.png
-   :align: center
-   :alt: Meal voucher
+It is possible to compare the current :guilabel:`Work entry analysis` report to the previous month
+or the previous year. To view these comparisons, click the :guilabel:`drop-down arrow` in the search
```

Comment:
i'd suggest using the arrow emoji mentioned above here, as it's featured in the Style Guide